apcd-cms/src/apps/submission_form/templates/submission_form/submission_form.html

```diff
@@ -0,0 +1,2968 @@
+        const fieldWrapper = fieldLabel.closest(".field-wrapper");
+        fieldWrapper.id = fieldLabel.getAttribute("for") + "--wrapper";
+      });
+
+      /* Wrap elements */
+      const startWith = '.field-wrapper[id^="contact_type"]';
+      const endBefore = `
+			hr, h6, h5, h4, h3, h2, h1,
+			.field-wrapper[id^="contact_role"],
+			.field-wrapper[id^="recaptcha"]
+		`;
+      const wrapWith = "div";
+      const wrapStarter = true;
+      wrapElements(startWith, endBefore, wrapWith, { wrapStarter, scope });
+
+      /* Style elements wrapper */
+      scope = getForm();
+      const elementsWrappers = scope.querySelectorAll(
+        ".js-wrapElements-wrapper:has(" + startWith + ")"
+      );
+      elementsWrappers.forEach((elementsWrapper) => {
+        elementsWrapper.classList.add("js-contact--wrapper");
+      });
+    </script>
+    <style id="form-wrap-contact-fields--css">
+      .js-contact--wrapper {
+        --max-col-width: 50ch;
+
+        /* Expectations
+				- automatically enough columns
+				- maximum column count of 2 (i.e. minimum column width of 50%)
+				- maximum column width of 50 char's (same as max-width of fields)
+				- equal width columns
+			*/
+        display: grid;
+        column-gap: var(--global-space--grid-gap);
+        grid-template-columns: repeat(auto-fill, var(--max-col-width));
+        max-width: calc(
+          var(--global-space--grid-gap) + var(--max-col-width) * 2
+        );
+      }
+      .field-wrapper[id^="contact"] {
+        min-height: 0;
+        grid-row: span 1;
+      }
+      .field-wrapper[id^="contact_email"] {
+        /* To mimic site.css `.help-text` margin-top */
+        margin-bottom: calc(var(--global-font-size--small) * 0.3);
+      }
+      .field-wrapper[id^="contact_phone"] {
+        /* HELP: span 2 is logical; span 3 raises _notifications */
+        grid-row: span 3;
+      }
+    </style>
+
+    <script id="form-file-submission--js" type="module">
+      const inputs = Array.from(
+        document.querySelectorAll(`
+        input[name=types_of_files_eligibility_enrollment],
+        input[name=types_of_files_provider],
+        input[name=types_of_files_medical],
+        input[name=types_of_files_pharmacy],
+        input[name=types_of_files_dental]`)
+      );
+      const inputListener = e => {
+      inputs.filter(i => i !== e.target)
+            .forEach(i => (i.required = !e.target.checked));
+      };
+
+      inputs.forEach(i => i.addEventListener('input', inputListener));
+    </script>
+
+    <script id="form-fein--js" type="module">
+      const inputs = Array.from(
+        document.querySelectorAll('input[name=fein], input[name=license_number], input[name=naic_company_code]')
+      );
+      const inputListener = e => {
+      inputs.filter(i => i !== e.target)
+            .forEach(i => (i.required = !e.target.value.length));
+      };
+
+      inputs.forEach(i => i.addEventListener('input', inputListener));
+    </script>
```

Comment:
**Excellent!** Thanks!

_Wes, this is what was necessary to fix the required multi-checkbox the right way. See if you can modularize/abstract this for other forms in the future._